projects/VS2022/raylib/raylib.vcxproj

```diff
@@ -302,6 +302,76 @@
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
+    <ClCompile Include="..\..\..\src\platforms\rcore_android.c">
```

Comment:
They are not required. Having them in the solution and marked as non buildable simply makes them easy to find and edit from within the IDE, instead of having to go find the files. It's purely a convenience feature for working in visual studio.
I can remove them if desired.